docs/standard/using-linq.md

```diff
@@ -155,7 +155,7 @@ The answer to this question is **no** if...
 
 ## Essential Samples
 
-For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](https://code.msdn.microsoft.com/101-LINQ-Samples-3fb9811b).
+For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](~/samples/dotnet/try-samples/101-linq-samples/).
```

Comment:
The link is broken. It resolves as https://review.docs.microsoft.com/en-us/dotnet/standard/~/samples/dotnet/try-samples/101-linq-samples/?branch=master.

```suggestion
For a truly comprehensive list of LINQ samples, visit [101 LINQ Samples](/samples/dotnet/try-samples/101-linq-samples/).
```

Reply:
I thought that the `~` was the correct syntax to change doc repos? I was wrong, great catch -- thank you. I kind of wish there was a preview version of the site, to catch these types of mistakes. How did you come about that link from above?

Reply:
There is a preview. Also, there was a warning in the build, but because we don't have build comments enabled in this repo, the build warnings are a bit hidden. Some people want to turn build comments on, and others don't. To see both the preview and the build warnings, click "Show all checks" near the bottom of the PR and then click "Details" next to OpenPublishing.Build.

As far as making the link work without the https://docs.microsoft.com prefix, you basically just chop that and the culture info off and it will work. But the only reason to do that is if the docset you're linking to is published as an offline book. That includes Windows. SQL, .NET, Visual Studio, EF (I think), ASP.NET.